packages/wp-now/src/constants.ts

```diff
@@ -0,0 +1,43 @@
+import os from 'os';
+import path from 'path';
+
+/**
+ * The hidden folder name for storing WP Now related files.
+ */
+export const WP_NOW_HIDDEN_FOLDER = '.wp-now';
+
+/**
+ * The full path to the hidden WP Now folder in the user's home directory.
+ */
+export const WP_NOW_PATH = path.join(os.homedir(), WP_NOW_HIDDEN_FOLDER);
+
+/**
+ * The path where WordPress zip files and unzipped folders will be stored within the WP Now folder.
+ */
+export const WORDPRESS_ZIPS_PATH = path.join(WP_NOW_PATH, 'wordpress_zips');
+
+/**
+ * The file name for the SQLite plugin name.
+ */
+export const SQLITE_FILENAME = 'sqlite-database-integration';
+
+/**
+ * The full path to the SQLite database integration folder.
```

Comment:
I think this is the path to where the database will be stored?
"database integration folder" is confusing me, which could be my lack of knowledge shining through.

Reply:
The folder is just the raw plugin: https://wordpress.org/plugins/sqlite-database-integration/
We save the SQLite database in `~/.wp-now/wp-content/${projectName}/database/ht.sqlite`

I've added the quotes to the plugin name as suggested in the following comment.
https://github.com/WordPress/wordpress-playground/pull/188/commits/1504b5f74767aff0b4aa7e1719fe2d224ae48e93